src/style.css

```diff
@@ -1,3 +1,75 @@
+:root {
+  --primary-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: auto;
+  background-color: white;
+  padding: 0 50px;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+}
+
+nav a {
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  font-weight: 500;
+  position: relative;
+  height: 100%;
+  display: inline-block;
+  line-height: 60px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. It's better to apply a class to these anchor elements and style them using that class.